src/Datasource/EntityTrait.php

```diff
@@ -1231,7 +1231,7 @@ public function source($alias = null)
      */
     public function __toString()
     {
-        return json_encode($this, JSON_PRETTY_PRINT);
+        return json_encode($this, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE);
```

Comment:
Update:
```
       $object = is_object($data);

        if ($object && ($data instanceof EntityTrait || $data instanceof JsonSerializable)) {
            return json_encode($data, JSON_UNESCAPED_UNICODE);
        }

        if ($object && method_exists($data, '__toString')) {
            return (string)$data;
        }
```

This changes the logging behaviour for everyone who does something like `$logger->log('debug', $entity)` with a custom logging format implemented by overwriting `EntityTrait::__toString()`. Don't you think that might take one or the other by surprise?